core/src/storage2/collections/binary_heap/mod.rs

```diff
@@ -269,3 +278,170 @@ where
         value
     }
 }
+
+/// An iterator over shared references to the elements of a storage vector.
```

Comment:
Rewrite this and other comments to mention that it is an iterator for a storage vector of `Group<T>` for binary heaps